devlib/target.py

```diff
@@ -279,6 +280,11 @@ def shutils(self):
             self._setup_shutils()
         return self._shutils
 
+    def is_running(self, process):
```

Comment:
nit: s/process/comm/. "process" could mean either PID or comm, but here it's specifically comm that we expect, we are not flexible on the input.